trie/resolver_stateful_cached.go

```diff
@@ -251,7 +251,20 @@ func (tr *ResolverStatefulCached) Walker(isAccount bool, blockNr uint64, fromCac
 		tr.curr.Write(tr.succ.Bytes())
 		tr.succ.Reset()
 		skip := tr.currentReq.extResolvePos // how many first nibbles to skip
-		tr.succ.Write(kAsNibbles[skip:])
+
+		//tr.succ.Write(kAsNibbles[skip:])
```

Comment:
1) the comment seems to be a leftover

2) we have `type hexWriter struct {` in `core/types/derive_sha.go`. We might want to make it public and just reuse it here.

Reply:
`hexWriter` - can't because here we may need to write even amount of nibbles.